src/components/Person/Person.jsx

```diff
@@ -1 +1,49 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age
+        ? <p className="Person__age">{`My age is ${age}`}</p>
+        : undefined}
```

Comment:
U can also use here logical AND, cuz you have only one option - if the age parameter is provided)